### PR TITLE
Add missing Elite and EDDB IDs

### PR DESCRIPTION
**Pull request by cmdrmcdonald**
There are a few missing Elite and EDDB iDs in master, I've added those where I have the data.

At this stage the only missing Elite IDs are those from the remaining PowerPlay modules.  I'm  hoping to catch those in EDDI over the next few weeks as people start to use it more, and will pass them along as I find them

### CODE REVIEW

File: modules/hardpoints/mine_launcher.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "nl": [
     { "id": "2j", "edID": 128049500, "eddbID": 880, "grp": "nl", "class": 1, "rating": "I", "cost": 24260, "mass": 2, "power": 0.4, "mount": "F", "type": "E", "armourpen": "C", "thermload": 2, "clip": 1, "ammo": 24, "ammocost": 209 },
     { "id": "2k", "edID": 128049501, "eddbID": 881, "grp": "nl", "class": 2, "rating": "I", "cost": 294080, "mass": 4, "power": 0.4, "mount": "F", "type": "E", "armourpen": "C", "thermload": 3, "clip": 3, "ammo": 24, "ammocost": 209 },
-    { "id": "kp", "eddbID": 0, "grp": "nl", "name":"Shock Mine Launcher", "class": 1, "rating": "I", "cost": 36400, "mass": 2, "power": 0.4, "mount": "F", "type": "E", "armourpen": "C", "thermload": 2, "clip": 1, "ammo": 24, "ammocost": 209 }
+    { "id": "kp", "edID: 128671448, "eddbID": 1523, "grp": "nl", "name":"Shock Mine Launcher", "class": 1, "rating": "I", "cost": 36400, "mass": 2, "power": 0.4, "mount": "F", "type": "E", "armourpen": "C", "thermload": 2, "clip": 1, "ammo": 24, "ammocost": 209 }
```
Comment (cmmcleod) [inline modules/hardpoints/mine_launcher.json]: Your are missing a `"` after `"edID`

File: modules/internal/pristmatic_shield_generator.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    { "id": "p2", "edID": 128671327, "eddbID": 1489, "grp": "psg", "class": 5, "rating": "A", "cost": 7655930,   "mass": 40,  "power": 5.46, "minmass": 203, "optmass": 405,  "maxmass": 1013, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 1.875 },
+    { "id": "p1", "edID": 128671328, "eddbID": 1490, "grp": "psg", "class": 6, "rating": "A", "cost": 24269300,  "mass": 80,  "power": 6.51, "minmass": 270, "optmass": 540,  "maxmass": 1350, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 2.4 },
+    { "id": "p0", "edID": 128671329, "eddbID": 1491, "grp": "psg", "class": 7, "rating": "A", "cost": 76933670,  "mass": 160, "power": 7.35, "minmass": 530, "optmass": 1060, "maxmass": 2650, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 3 },
+    { "id": "p7", "edID": 202115326, "eddbID": 1492, "grp": "psg", "class": 8, "rating": "A", "cost": 243879730, "mass": 320, "power": 8.4,  "minmass": 900, "optmass": 1800, "maxmass": 4500, "minmul": 2, "optmul": 1.5, "maxmul": 1, "recover": 3.6 }
```
Comment (Marginal) [inline modules/internal/pristmatic_shield_generator.json:10]: Are you sure about this value? 202115326 is out of sequence - I would have expected 128671330.

Comment (cmmcleod) [inline modules/internal/pristmatic_shield_generator.json:10]: @cmdrmcdonald If you can't verify I will try to do so tonight.

**Comment by cmdrmcdonald** [comment]
You're right.  I managed to cut 'n' paste the module's value rather than its ID for that one.  Apologies.

**Pull request merged**